ROADMAP.md

```diff
@@ -1,8 +1,19 @@
 # Roadmap
 
+## An even better tool for development environments
+
+Compose is a great tool for development environments, but it could be even better. For example:
+
+- [Compose could watch your code and automatically kick off builds when something changes.](https://github.com/docker/fig/issues/184)
```

Comment:
Could we discuss dropping this from the roadmap? It doesn't feel appropriate for compose. It's trivial to do this with an external tool that watches files and calls build/up when one changes. We could just document that, instead of building it in.